pype/scripts/transcode.py

```diff
@@ -0,0 +1,176 @@
+"""
+transcode.py -input "input.%04d.exr [1-77]" -name name
+-preset prores422HQ review -output path/to/output/folder
+
+transcode.py -input input.mov -name name
+-preset prores422HQ review -output path/to/output/folder
+
+transcode.py -input input.txt -name name
+-preset concat -output path/to/output/folder
+
+Concatenation *.txt files follows syntax from the FFMPEG demuxer guide;
+https://trac.ffmpeg.org/wiki/Concatenate
+"""
+import os
+import sys
+import argparse
+import tempfile
+
+import clique
+from pype import lib
+
+
+preset_templates = {
+    "prores422HQ": {
+        "args": [
+            "-c:v", "prores_ks",
+            "-profile:v", "3",
+            "-pix_fmt", "yuv422p10le",
+            "-qscale:v", "5",
+            "-codec:a", "pcm_s16le"
+        ],
+        "extension": ".mov"
+    },
+    "proresProxy": {
+        "args": [
+            "-c:v", "prores_ks",
+            "-profile:v", "0",
+            "-pix_fmt", "yuv422p10le",
+            "-qscale:v", "5",
+            "-codec:a", "pcm_s16le",
+            "-vf", "scale=trunc(oh*a/2)*2:trunc(ih/2)"
+        ],
+        "extension": ".mov"
+    },
+    "h264": {
+        "args": ["-crf", "18", "-pix_fmt", "yuv420p"],
+        "extension": ".mov"
+    },
+    "concat": {
+        "args": ["-c", "copy"],
+        "extension": ".mov"
+    }
+}
+
+
+def main(input_path="",
+         preset="",
+         name="",
+         output_path="",
+         audio_path="",
+         start=None,
+         framerate=None):
+
+    collection = None
+    try:
+        collection = clique.parse(input_path)
+    except ValueError:
+        pass
+
+    first_file = input_path
+    if collection:
+        first_file = list(collection)[0]
+
+    output_path = os.path.join(
```

Comment:
did you mean this? 
```
output_path = output_path or os.path.join(
    output_path, name + preset_templates[preset]["extension"]
).replace("//", "/")
```

Reply:
See above comment on `output_path` argument.